docs/extras/bootstrap.md

```diff
@@ -61,7 +61,7 @@ The `bootstrap_nav.*` templates produce the same output, and can be used as an e
 
 === `pagy_bootstrap_nav_js(pagy, ...)`
 
-![bootstrap_nav_js - Notice how it gets shorter, depending on width?](/docs/assets/images/bootstrap_nav_js-g.png)
+![bootstrap_nav_js: It is responsive.](/docs/assets/images/bootstrap_nav_js-g.png)
```

Comment:
Then I would prefer "Responsive nav".

Reply:
... and no need to force-push, because Github can do it by itself on merge.

Reply:
fixed with: [3605d15](https://github.com/ddnexus/pagy/pull/453/commits/3605d157420478e68f41fa50711f1950adadfc5a)